shared-helpers/src/locales/general.json

```diff
@@ -1014,6 +1014,9 @@
   "welcome.findAffordableHousing": "Find affordable housing in the Bay Area",
   "welcome.useDoorway": "Use the Doorway Housing Portal to apply for affordable housing across the Bay Area. Getting housing is hard. Doorway is working to make it easier.",
   "welcome.learnHowToUseDoorway": "Learn how to use the portal",
+  "welcome.peopleHoldingHandsAlt": "Hands of caregivers and elderly people in wheelchairs",
+  "welcome.personLaptopAlt": "Shot of a young woman using a laptop and going through paperwork while working from home",
+  "welcome.personWithChildAlt": "Shot of a young man carrying his daughter on his shoulders",
```

Comment:
Have you been able to test the reader/audio on this? I believe it already says "photo of" before the Alt Text, so you may want to remove "shot of" on the latter 2 items here.

Reply:
oh i didn't know that thanks for the heads up! yeah i juts grabbed this text from the metadata on the stock photos, will update.

![image](https://github.com/metrotranscom/doorway/assets/10789523/816a6ed1-f6e0-4801-960b-1741d62e420f)